kubectl-cloudflow/domain/cloudflowapplication.go

```diff
@@ -194,36 +194,38 @@ func ApiVersion() string {
 }
 
 // UpdateCloudflowApplication creates a CloudflowApplication struct that can be used with a Update call
-func UpdateCloudflowApplication(spec CloudflowApplicationSpec, resourceVersion string) CloudflowApplication {
+func UpdateCloudflowApplication(spec CloudflowApplicationSpec, currentAppCR CloudflowApplication, releaseTag string, buildNumber string) CloudflowApplication {
```

Comment:
for consistency, `currentAppCR` could be called `currentApp` xor `UpdateCloudflowApplication` should be `UpdateCloudflowApplicationCR` but I think overall it's called `CloudflowApplication` so I'd go with the former.